### PR TITLE
Visualize objects in bin on Euslisp

### PR DESCRIPTION
**Pull request by wkentaro**
image

### CODE REVIEW

**Comment by wkentaro** [comment]
@knorth55 Please see the change about euslint is ok for you?

**Comment by knorth55** [comment]
@wkentaro what is your purpose?
In euslint, it already have `has_quote` flag, but what is your case?

**Comment by wkentaro** [comment]
I had error below:
```
% rosrun jsk_2016_01_baxter_apc euslint samples/sample_eus_visualize_objs.l
parenthesis: Number of open/close parenthesises don't much: 55 != 54
```

because the file has `(warn ";; XXX")`

**Comment by wkentaro** [comment]
I have another error, 

```
% rosrun jsk_2016_01_baxter_apc euslint euslisp/jsk_2016_01_baxter_apc/util.l
whitespace: Invalid white space usage at line 10
```

How can I fix it? @knorth55

**Comment by knorth55** [comment]
@wkentaro Fixed and cherry-pick the commit. https://github.com/wkentaro/jsk_apc/pull/7

**Comment by knorth55** [comment]
@wkentaro sorry, but i updated the commit. please cherry-pick again :( 
https://github.com/knorth55/jsk_apc/commit/11bc5f833c7d7e6998dd706839592322bfc4a99a

**Comment by wkentaro** [comment]
picked.

**Comment by knorth55** [comment]
thx

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 1]
```diff
@@ -438,19 +438,31 @@
     ()
     (let (bins)
       (dolist (b (send _bin-boxes :list-values))
-        (pushback (send self :bbox->cube b) bins)
-        )
-      (objects (append (list *baxter*) bins))
-      ))
+        (let* ((bin-inside (send self :bbox->cube b))
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: in this case, `let` is ok

File: jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+(defmethod jsk_2016_01_baxter_apc::baxter-interface
+  (:graspingp (arm) t)
+  )
+
+
+(jsk_2016_01_baxter_apc::baxter-init)
+(setq *irtviewer* (make-irtviewer))
+(send *irtviewer* :change-background #f(0.8 0.8 0.8))
+
+(setq right_gripper_bodies nil)
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l]: naming rule for variables is concat words with `-` not `_`.

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l]: I'm changing it. But using vim, I don't have the completion about variable name when I use `-`, how do you fix it?

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l]: I found the solution: http://qiita.com/YamasakiKenta/items/fee2a366348b9da345e8

Comment (knorth55) [inline jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l]: I mean all variables name should be concated with  `-`.

Comment (knorth55) [inline jsk_2016_01_baxter_apc/samples/sample_eus_visualize_objs.l]: `all_bodies`, `bin_box_msg` etc.

**Pull request merged**